docs/modules/casper.rst

```diff
@@ -1359,16 +1370,21 @@ Retrieves current page title::
 ``mouseEvent()``
 -------------------------------------------------------------------------------
 
-**Signature:** ``mouseEvent(String type, String selector)``
+**Signature:** ``mouseEvent(String type, String selector, [Number|String X, Number|String Y])``
 
 .. versionadded:: 0.6.9
+.. versionupdate:: 1.1.0-beta6
 
 Triggers a mouse event on the first element found matching the provided selector.
 
-Supported events are ``mouseup``, ``mousedown``, ``click``, ``mousemove``, ``mouseover`` and ``mouseout``::
+Supported events are ``mouseup``, ``mousedown``, ``click``, ``dblclick``, ``mousemove``, ``mouseover``, ``mouseout``
+and for phantomjs >= 1.9.8 ``mouseenter``, ``mouseleave`` and ``contextmenu``::
+
+.. warning::
+Events depend on version Browser - to generate all kinds of Popular Events use the latest phantomJS or SlimerJS versions
```

Comment:
"The list of supported events depends on the version of the engine in use. Older engines only provide partial support. For best support use recent builds of PhantomJS or SlimerJS."